### PR TITLE
302 redirect detection

### PR DESCRIPTION
**Pull request by rgc**
Adding myself to Authors/Contrib following CLA submission

### CODE REVIEW

File: lib/util/failover_uri.js [hunk 1]
```diff
@@ -168,6 +168,9 @@ shaka.util.FailoverUri.prototype.createRequest_ =
         // effectively cache every response.
         this.requestPromise_ = null;
         this.request_ = null;
+	if(xhr.responseURL) {
```
Comment (joeyparrish) [inline lib/util/failover_uri.js]: This whole section appears to be mis-indented. Please run the linter (./build/lint.sh) and compiler (./build/build.sh) and make sure that all checks pass before submitting a PR.

Comment (rgc) [inline lib/util/failover_uri.js]: I'm fixing with a different pull request, which includes the information you needed and has had lint.sh run

File: lib/util/failover_uri.js [hunk 2]
```diff
@@ -168,6 +168,9 @@ shaka.util.FailoverUri.prototype.createRequest_ =
...
+		this.urls.unshift(new goog.Uri(xhr.responseURL));
```
Comment (joeyparrish) [inline lib/util/failover_uri.js]: What would be better than prepending the URL would be to replace the specific URL this request used.  (this.urls[i])

**Comment by joeyparrish** [comment]
Ah, I see this is out-of-date, and you've already sent me #266.  Closing.

**Pull request closed without merging**